### PR TITLE
Develop global reanalysis discovery for EIS thematic area

### PR DESCRIPTION
**Pull request by KathyNie**
## Why are you creating this Pull Request?

Converting the global TWS trend discovery into a global reanalysis discovery.

*** Please don't merge this pull request to the main branch yet, issuing this pull request for preview purposes internally ***

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -8,10 +8,10 @@ APP_DESCRIPTION=Visualization, Exploration, and Data Analysis (VEDA)
 APP_CONTACT_EMAIL=email@example.org
 
 # Endpoint for the Tiler server. No trailing slash.
-API_RASTER_ENDPOINT='https://staging-raster.delta-backend.com'
+API_RASTER_ENDPOINT='https://dev-raster.delta-backend.com'
```
Comment (hanbyul-here) [inline .env:11]: Can you change this back to staging?

File: .veda/ui
Comment (hanbyul-here) [inline .veda/ui:1]: can you exclude ui module from this pr?

File: .env (hunk 1 above)
Comment (KathyNie) [reply .env]: Changed!

File: .veda/ui
Comment (KathyNie) [reply .veda/ui:1]: Hmm... Do you have any suggestions on how to exclude ui module change? I didn't explicitly work on anything with respect to this directory. I'm a bit confused why these changes are made.

**Comment by danielfdsilva** [comment]
@KathyNie Is everything else ready on this PR? To facilitate things, we can take care of fixing the UI module conflict.

**Comment by danielfdsilva** [comment]
Content squashed, and conflicts cleared.
Added through this commit https://github.com/NASA-IMPACT/veda-config/commit/7105b3d9f958fe30adb054f6a91246277cb310d1

**Comment by KathyNie** [comment]
@danielfdsilva Hi Daniel, I have a few minor errors that need to be fixed for this discovery and the associated dataset. I wonder if I should open a new pull request for modification or is there a way to reopen this ticket and continue to work on it? Any suggestions?

**Comment by danielfdsilva** [comment]
Hello @KathyNie 👋 
The best and easiest course of action would be to open a new PR (branching from `develop`) with your proposed changes. If you hit a snag let me know.

**Pull request closed without merging**